Rules.py

```diff
@@ -6,20 +6,32 @@ def set_rules(world, player):
 
     if world.logic == 'nologic':
         logging.getLogger('').info('WARNING! Seeds generated under this logic often require major glitches and may be impossible!')
-        world.get_region('Links House', player).can_reach = lambda state: True
-        world.get_region('Sanctuary', player).can_reach = lambda state: True
-        old_rule = world.get_region('Old Man House', player).can_reach
-        world.get_region('Old Man House', player).can_reach = lambda state: state.can_reach('Old Man', 'Location', player) or old_rule(state)
-        return
-
-    global_rules(world, player)
+        if world.mode != 'inverted':
+            world.get_region('Links House', player).can_reach = lambda state: True
+            world.get_region('Sanctuary', player).can_reach = lambda state: True
+            old_rule = world.get_region('Old Man House', player).can_reach
+            world.get_region('Old Man House', player).can_reach = lambda state: state.can_reach('Old Man', 'Location', player) or old_rule(state)
+            return
+        else:
+            world.get_region('Inverted Links House', player).can_reach = lambda state: True
+            world.get_region('Inverted Dark Sanctuary', player).entrances[0].parent_region.can_reach = lambda state: True
+            if world.shuffle != 'vanilla':
+                old_rule = world.get_region('Old Man House', player).can_reach
+                world.get_region('Old Man House', player).can_reach = lambda state: state.can_reach('Old Man', 'Location', player) or old_rule(state)
+            else:
+                world.get_region('Dark Death Mountain', player).can_reach = lambda state: state.can_reach('Old Man', 'Location', player)
```

Comment:
This looks to be missing a return statement. No logic should simply set the starting areas accessible and then return.

Reply:
Yeah I messed up here. I removed the conditional entirely. In vanilla, DDM should be covered by normal access checks with flute or gloves and lamp. When shuffled, it should be the same as non-inverted shuffles. Does that sound correct?

Reply:
Um.. Are you sure you are looking at the spot i was pointing to?

I was talking about the no-logic branch. at the very top of rules.py.

I think what you are saying might be correct for the spot you actually changed, but I've not thought about it too closely.

Reply:
Oh, whoops. Was looking at the code instead of the line numbers here

Reply:
Fixed